pkgs/by-name/di/dita-ot/package.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchzip
+, bash
+, jdk17
+, nix
+, coreutils
+, lib
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "dita-ot";
+  version = "4.2.1";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+  buildInputs = [
+    bash
+    jdk17
+  ];
+
+  src = fetchzip {
+    url = "https://github.com/dita-ot/dita-ot/releases/download/${finalAttrs.version}/dita-ot-${finalAttrs.version}.zip";
+    hash = "sha256-PtscKsQwW1uBu0OfVHi7/d9JujGx12nZT+W3uMPQQWc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/$name/
+    cp -r $src/* $out/$name/
+    chmod -R 777 $out/$name/
```

Comment:
This seems rather excessive. Does every file really need the executable bit?

Reply:
Apparently, `chmod` is not anymore required with `makeWrapper`.